src/NLU.DevOps.Luis/LuisPredictionClient.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace NLU.DevOps.Luis
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Logging;
+    using Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime;
+    using Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime.Models;
+    using Microsoft.CognitiveServices.Speech;
+    using Microsoft.CognitiveServices.Speech.Audio;
+    using Microsoft.CognitiveServices.Speech.Intent;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    internal sealed class LuisPredictionClient : ILuisPredictionClient, IDisposable
```

Comment:
While we're at it, should we also split into ILuisPredictionClient into a text and speech interface?

Reply:
Can do that